sky/clouds/gcp.py

```diff
@@ -916,6 +931,36 @@ def _check_instance_type_accelerators_combination(
             resources.instance_type, resources.accelerators, resources.zone,
             'gcp')
 
+    @classmethod
+    def check_disk_tier(
```

Comment:
This function seems not publicly exposed? Should we keep it private, or just merge into `check_disk_tier_enabled`?

Reply:
Good point! Converted it into private. merge into `check_disk_tier_enabled` is a little bit tricky as we want a function to return a boolean value, instead of raising an error, which is the expected behaviour of `check_disk_tier_enabled`, across all clouds.

Reply:
Actually, I found a bug that requires to expose this function. Just changed back to public.